Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+    agent any
+
+    stages {
+
+        stage("Clean") {
+            steps {
+                sh "./gradlew clean"
+            }
+        }
+
+        stage("Assemble") {
+            steps {
+                sh "./gradlew amp"
```

Comment:
`Assemble` != `amp`?

I'd expect that a `./gradlew assemble` builds the amp artifact as well?

Reply:
yes